tests/test_data_helpers.py

```diff
@@ -31,10 +31,25 @@ def obj_from_fixture(self, filename, client=None):
 
     def test_get_note_text(self):
         """Checks whether the returned note text matches the selected query string."""
-        note = self.obj_from_fixture("note_multi.json")
-        result = data_helpers.get_note_text(note)
-        self.assertTrue(result, list)
-        self.assertEqual(result, ["materials are restricted"])
+        for fixture, expected in [
+                ("note_bibliography.json", [
+                    "bibliography", "item 1", "item 2"]),
+                ("note_index.json", ["title1", "title2"]),
+                ("note_multi.json", ["materials are restricted"]),
+                ("note_multi_chronology.json", [
+                    "general note with chronology", "date", "event1", "event2"]),
+                ("note_multi_defined.json", [
+                    "bioghist with defined list", "item", "1", "item", "2"]),
+                ("note_multi_ordered.json", [
+                    "Bioghist with ordered list", "item1", "item2"]),
+                ("note_single.json", ["New York Mets"])]:
+            note = self.obj_from_fixture(fixture)
+            result = data_helpers.get_note_text(note)
+            print(result)
```

Comment:
```suggestion
            print(result)
```
Let's remove this line, eh?